halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/NodeFilter.java

```diff
@@ -64,6 +64,8 @@ public NodeFilter anyMaster() {
     return this;
   }
 
+  public NodeFilter() { }
```

Comment:
Do you mean to filter everything or did you really mean nothing?
Does this mean you need to explicitly populate everything to use [at least everything above some level]?